lib/miq_automation_engine/service_models/miq_ae_service_cloud_volume.rb

```diff
@@ -10,5 +10,7 @@ class MiqAeServiceCloudVolume < MiqAeServiceModelBase
     expose :cloud_volume_backups,   :association => true
     expose :cloud_volume_snapshots, :association => true
     expose :attachments,            :association => true
+    expose :attach_volume,          :override_return => nil
+    expose :detach_volume,          :override_return => nil
```

Comment:
Is it possible to return a success or failure status of some kind?  The user will have no indication if the method was successful if we always return `nil`.